ui_testing/testcases/header_tests/test08_usermanagement.py

```diff
@@ -85,78 +93,59 @@ def test004_download_user_sheet(self):
+        self.assertTrue(result, created_user)
         self.header_page.select_all_records()
-        self.header_page.click_on_user_right_menu()
-        self.header_page.click_on_delete_button()
+        self.base_selenium.LOGGER.info('Press on the right menu')
+        self.base_selenium.click(element='user_management:right_menu')
+        self.base_selenium.LOGGER.info('Press on the delete button')
+        self.base_selenium.click(element='user_management:delete')
         self.header_page.confirm_popup()
-        result = self.header_page.search(value=self.header_page.user_name)[0].text
+        result = self.header_page.search(value=random_user_name)
         self.base_selenium.LOGGER.info('deleted successfully')
-        self.assertFalse(result, 'deleted successfully')
+        self.assertTrue(result, 'No records found')
 
-    def test006_create_new_user(self):
+    def test006_create_new_user_with_admin_role(self):
         """
         Header: User management Approach:  Make sure that I can create new user successfully
         LIMS-6000
         :return:
         """
-        self.header_page.click_on_user_management_button()
+        self.base_selenium.click(element='header:user_management_button')
         #create new user
-        self.header_page.create_new_user(user_email=(self.header_page.generate_random_email()), user_role='',
+        random_user_name = self.generate_random_string()
+        user_data= self.header_page.create_new_user(user_name = random_user_name, user_email=(self.header_page.generate_random_email()), user_role='Admin',
                                          user_password='1', user_confirm_password='1')
 
         #make sure when you search you will find it
-        user_text = self.header_page.search(value=self.header_page.user_name)[0].text
-        self.assertIn(self.header_page.user_name, user_text)
-
-
-    def test007_user_filter(self):
-        """
-        Header: User management Approach: Make sure that you can filter by any field in the active table
-        LIMS-6002
-        :return:
-        """
-        self.header_page.click_on_user_management_button()
-        # create new user to get it's data
-        user = self.header_page.create_new_user(user_email=(self.header_page.generate_random_email()), user_role='',
-                                                user_password='1', user_confirm_password='1')
-        # the filter view will open
-        self.header_page.click_on_filter_view()
-
-        # filter by user name
-        self.header_page.filter_user_by(filter_element='user_management:filter_name',
-                                        filter_text=user['user_name'])
-        result_user = self.header_page.result_table()[0]
-        self.assertIn(user['user_name'], result_user.text)
-        self.header_page.filter_reset_btn()
+        result = self.header_page.search(value=random_user_name)
+        self.assertTrue(result, user_data)
```

Comment:
where is the assertion of the `admin` role?

Reply:
@0xIslamTaha why I assert with the admin role , I filtered by name